ui/src/views/storage/UploadLocalVolume.vue

```diff
@@ -103,12 +88,7 @@
           </a-select>
         </a-form-item>
         <a-form-item>
-          <span slot="label">
-            {{ $t('label.volumechecksum') }}
-            <a-tooltip :title="apiParams.checksum.description">
-              <a-icon type="info-circle" style="color: rgba(0,0,0,.45)" />
-            </a-tooltip>
-          </span>
+          <tooltip-label slot="label" :title="$t('label.volumechecksum')" :tooltip="apiParams.volumechecksum.description"/>
```

Comment:
```suggestion
          <tooltip-label slot="label" :title="$t('label.volumechecksum')" :tooltip="apiParams.checksum.description"/>
```

As the form fails to load with the following error
![image](https://user-images.githubusercontent.com/10495417/125560867-d54ceb22-fb0d-4726-9679-a886b40d0e0c.png)